nncf/tensorflow/graph/converter.py

```diff
@@ -760,11 +761,11 @@ def _get_conv_layer_attributes(layer: tf.keras.layers.Layer, is_depthwise: bool
                                       padding_values=([0, 0, 0, 0]))
 
 
-def _get_linear_layer_attributes(layer: tf.keras.layers.Layer) -> LinearLayerAttributes:
+def _get_linear_layer_attributes(layer: tf.keras.layers.Layer, layer_metatype) -> LinearLayerAttributes:
     channel_axis = get_input_channel_axis(layer)
     in_features = layer.get_input_shape_at(0)[channel_axis]
     out_features = layer.get_output_shape_at(0)[channel_axis]
-    bias = layer.use_bias
+    bias = False if layer_metatype == op_metatypes.TFMatMulOpMetatype else layer.use_bias
```

Comment:
For TFOpLambda layer, there's no `use_bias` property. Added a workaround. This is a regression after #1319.